README.md

```diff
@@ -359,6 +359,9 @@ export interface MetrikaCounter {
   trackHash?: boolean;
   ecommerce?: boolean | string;
   type?: number;
+  userParams?: {
+    [x: string]: boolean | string | number;
+  };
```

Comment:
Update?

Reply:
update readme and add `UserParams` in export